LibLancBlocking.h

```diff
@@ -42,32 +45,60 @@ class LancBlocking
    * @param far Whether to pusht the Focus farther away or closer
    */
   void Focus(bool far);
+
+  void RecordToggle();
   /**
-   * Toggle Autofocus
-   * 
-   * @note Maybe we could read whether the Autofocus is set by the return value 
+   * Autofocus
+   *
+   * @note Maybe we could read whether the Autofocus is set by the return value
    *       and thus change according to the response.
    */
   void AutoFocus();
   /**
-   * Set the next lanc transmission to no command
+   *  AutoIris
+   *
+   * @note This command is untested,  its the proper code to trigger the iris,
+   *       but the camera I had on hand did not support it.
+   */
+  void AutoIris();
+  /**
+   *  Iris
+   *
+   * @note This command is untested,  its the proper code to trigger the open/close
+   *       but the camera I had on hand did not support it.
+   */
+  void Iris(bool moreopen);
+  /**
+   *  ClearCommand
+   *
+   * @note Set the next lanc transmission to no command
    */
   void ClearCommand();
+  /**
+   *  PowerDown
+   *
+   * @note Instruct the camera to power down.  lanc communications will likley stop on the cameras part after this command completes.
+   */
+  void PowerDown();
+  void VideoCameraSpecialCommand(uint8_t data);
```

Comment:
Can you please add comments to these public methods? What do you need them for? If you intend to use them as a possibility for further expansion, documetnation will be essential here. Also I'm missing a second parameter here if I do understand the purpose of these methods correctly....